environments/openshift/main.jsonnet

```diff
@@ -13,13 +14,15 @@ local app =
       list.asList('observatorium', {}, []) + {
         objects:
           $.thanos.template.objects +
-          $.telemeterServer.list.objects,
+          $.telemeterServer.list.objects +
+          $.prometheus.template.objects,
```

Comment:
We should name this something way more specific to describe what that Prometheus is for.

Reply:
`amsPrometheus`? `subscriptionPrometheus`? `subcriptionSyncer` ? 
What would be the most descriptive?